src/steps/ecs/client/types/response.ts

```diff
@@ -0,0 +1,13 @@
+import { Instance } from '../../types';
+import { PaginatedResponse } from '../../../../client/types/response';
+
+export interface DescribeInstancesResponse extends PaginatedResponse {
+  Instances: {
+    Instance: Instance[];
+  };
+  PageNumber: number;
+  PageSize: number;
+  TotalCount: number;
+  RequestId: string;
+  NextToken: string;
+}
```

Comment:
nit: since this is extending `PaginatedResponse`, couldn't you do away with the base properties? I may be missing something here though. 

```suggestion
export interface DescribeInstancesResponse extends PaginatedResponse {
  Instances: {
    Instance: Instance[];
  };
}
```

Reply:
Whoops, yes you are right; those base properties shouldn't be there. Good catch!